bagel/utilities/file_utils.py

```diff
@@ -95,8 +95,8 @@ def check_overwrite(output: Path, overwrite: bool):
 
 
 def save_jsonld(data: dict, filename: Path):
-    with open(filename, "w") as f:
-        f.write(json.dumps(data, indent=2))
+    with open(filename, "w", encoding="utf-8") as f:
+        f.write(json.dumps(data, indent=2, ensure_ascii=False))
```

Comment:
is this in response to a bug? If so, we should probably add a test for whatever the bug was about

Reply:
Not a bug, I just noticed when testing the output that currently non-ascii characters like é are escaped (`\u00e9`) in the final jsonld. Afaik this shouldn't have any functional consequence on downstream tools since the characters are decoded into Unicode again as soon as they're read in by Python (e.g., in n-API responses) but they're just less human-readable in the raw file.